akka-sample-fsm-java/src/main/java/sample/Chopstick.java

```diff
@@ -0,0 +1,125 @@
+package sample;
+
+import akka.actor.typed.ActorRef;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+
+public class Chopstick {
+    interface ChopstickMessage { }
```

Comment:
repeating `Chopstick` in this name is probably not needed since the messages should be used from the outside of the actor as `Chopstick.Message`, `Chopstick.Take`

Reply:
Also all messages are commands, so maybe make it `Command`